test/test_suite.py

```diff
@@ -1210,3 +1210,110 @@ def test_limit_offset_selectable_in_unions(self):
     )
     def test_order_by_selectable_in_unions(self):
         pass
+
+
+class JoinTest(fixtures.TablesTest):
```

Comment:
Why are we reimplementing this test class rather than overloading it?

Reply:
Aren't they part of `test_select`: https://github.com/sqlalchemy/sqlalchemy/blob/master/lib/sqlalchemy/testing/suite/test_select.py#L625-L722